apps/export/models.py

```diff
@@ -75,12 +75,14 @@ class Export(models.Model):
 
     pending = models.BooleanField(default=True)
     status = models.CharField(max_length=30, choices=STATUS_CHOICES, default=PENDING)
+    is_deleted = models.BooleanField(default=False)
 
     def __str__(self):
         return self.title
 
     @staticmethod
     def get_for(user):
         return Export.objects.filter(
-            exported_by=user
+            exported_by=user,
+            is_deleted=False
```

Comment:
@AdityaKhatri Do you think we also need to filter access by project permission? If a user is not a member of a project they don't have access to their exports from that project?